image_publisher.sh

```diff
@@ -13,8 +13,10 @@ DOCKER_HUB_ACCOUNT=thyrlian
 DOCKER_FILE_NAME=Dockerfile
 MAIN_IMAGE_NAME=android-sdk
 MAIN_IMAGE_DIR=android-sdk
-SUB_IMAGE_NAME=android-sdk-vnc
-SUB_IMAGE_DIR=vnc
+SUB_IMAGE_VNC_NAME=android-sdk-vnc
+SUB_IMAGE_VNC_DIR=vnc
+SUB_IMAGE_FIREBASE_TEST_LAB_NAME=android-firebase-test-lab
+SUB_IMAGE_FIREBASE_TEST_LAB_DIR=firebase_test_lab
```

Comment:
Following the naming convention, the directory name should use hyphen instead of underscore: `firebase-test-lab`